pkgs/by-name/so/sold/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sold";
+  version = "1.11.0";
+
+  src = fetchFromGitHub {
+    owner = "bluewhalesystems";
+    repo = pname;
+    rev = "a52e8b37661562c866a579ab1164f94694f479a5";
+    sha256 = "0iyrgz0ay2453rqrlvxsrla0998l8yq6q3xyqqw5icfvh05ld8i8";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  enableParallelBuilding = true;
+
+  meta = {
+    description = "A fork of mold, a faster drop-in replacement for existing Unix linkers, with support for iOS and MacOS";
+    license = lib.licenses.mit;
+    homepage = "https://github.com/bluewhalesystems/sold";
+    maintainers = with lib.maintainers; [ elise ];
+    platforms = lib.platforms.all;
```

Comment:
`mainProgram` is missing.

Reply:
Thank you, I think I have resolved this, I placed what I feel would be the most appropriate binary there